### PR TITLE
Remove create command 

### LINKED ISSUE #300: Remove the create command from the palette
The `create` command doesn't need to be shown in the palette for now, since the ability to close the sidebar was removed in #298.

image

### PR DESCRIPTION
**Pull request by KsavinN**
resolve #300 

- [x] remove create command from the palette.

- [x] remove command create and create sidebar directly.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -421,52 +419,43 @@ const main: JupyterFrontEndPlugin<IDebugger> = {
...
-        });
-
-        sidebar.service.sessionChanged.connect(_ => {
-          commands.notifyCommandChanged();
-        });
-
-        if (labShell.currentWidget) {
-          labShell.currentWidget.activate();
-        }
+    // Createing sidebar instance and add to shell
```
Comment (jtpio) [inline src/index.ts]: It's probably also fine without a comment here.

File: src/index.ts [hunk 2]
```diff
@@ -421,52 +419,43 @@ const main: JupyterFrontEndPlugin<IDebugger> = {
...
 
-        shell.add(sidebar, 'right');
-      }
+    sidebar.service.sessionChanged.connect(_ => {
+      commands.notifyCommandChanged();
     });
 
+    if (labShell.currentWidget) {
+      labShell.currentWidget.activate();
+    }
```
Comment (jtpio) [inline src/index.ts]: Wondering whether we still need this actually, or if it can be removed?

File: src/index.ts [hunk 3]
```diff
@@ -421,52 +419,38 @@ const main: JupyterFrontEndPlugin<IDebugger> = {
...
-        });
-
-        sidebar.service.sessionChanged.connect(_ => {
-          commands.notifyCommandChanged();
-        });
-
-        if (labShell.currentWidget) {
-          labShell.currentWidget.activate();
-        }
+    sidebar = new Debugger.Sidebar({
```
Comment (jtpio) [inline src/index.ts]: One last thing: we could declare and define the `sidebar` variable here directly, with:

```typescript
const sidebar = new Debugger.Sidebar({...})
```

Comment (KsavinN) [reply src/index.ts]: rigth

**Pull request merged**